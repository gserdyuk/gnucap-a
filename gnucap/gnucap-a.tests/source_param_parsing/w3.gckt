* sources with dc, ac and pwl, ids without quotes; ac with two params

.param a=1 b=2 angle=90
v1 1 0 dc a ac b angle pwl 0 0 10ns 5 15ns 2.5

r1 1 0 10k


.print op v(*) i(V*)
.op

.print ac v(*) i(V*)
.ac dec 2 1 10

.print tran v(*) i(V*)
.tran 0 20ns 1ns

.end
