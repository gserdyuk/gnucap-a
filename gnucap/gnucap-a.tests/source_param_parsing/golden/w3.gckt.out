Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-09-25  bmm_semi_hsp, bm_model fix, back to bmm_semi
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* sources with dc, ac and pwl, ids without quotes; ac with two params 
#           v(1)       v(v1)      v(r1)      i(v1)     
 27.        1.         1.         1.        -100.u     
#Freq       v(1)       v(v1)      v(r1)      i(v1)     
 1.         2.         2.         2.         200.u     
 3.1623     2.         2.         2.         200.u     
 10.        2.         2.         2.         200.u     
